frontend/src/stores/DataStore.js

```diff
@@ -227,6 +227,13 @@ class DataStore {
     @computed get getDatasetNamesHeader() {
         return datasetNamesHeaders[this.getDatasetType];
     }
+
+    @computed get checkDatasetsLength() {
```

Comment:
same comment as below; change to `canAddNewDataset`, flip logic, and make a oneliner